backend/app/services/channels/dingtalk/sender.py

```
@@ -204,3 +207,565 @@ async def _send_message(
+                return {
+                    "success": True,
+                    "outTrackId": out_track_id,
+                    "result": data,
+                }
+
+        except httpx.HTTPStatusError as e:
+            error_data = {}
+            try:
+                error_data = e.response.json()
+            except Exception:
+                pass
```

Ruff (0.15.6)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)